sql/scan.go

```diff
@@ -682,6 +683,13 @@ func prettyKeyVals(vals []parser.Datum) string {
 			fmt.Fprintf(&buf, "/%v", t)
 		case parser.DString:
 			fmt.Fprintf(&buf, "/%v", t)
+		case parser.DDate:
```

Comment:
There was a point at one time in the code's history, but not any longer. The entire loop body could be simplified to `fmt.Fprintf(&buf, "/%v", v)`.